daemon/daemon.go

```diff
@@ -967,8 +967,9 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					if err := c.KillSig(15); err != nil {
-						logrus.Debugf("kill 15 error for %s - %s", c.ID, err)
+					// If container failed to exit in 10 seconds of SIGTERM, then using the force
+					if err := c.Stop(10); err != nil {
+						logrus.Debugf("Stop container %s with error: %s", c.ID, err.Error())
```

Comment:
Let's print `Errorf` here, so people can trace why their process died. Also, you should use `%v` for err and use just `err` without `.Error`